src/ci/docker/scripts/sccache.sh

```diff
@@ -1,6 +1,16 @@
 set -ex
 
-curl -fo /usr/local/bin/sccache \
-  https://ci-mirrors.rust-lang.org/rustc/2018-04-02-sccache-x86_64-unknown-linux-musl
+case "$(uname -m)" in
+    x86_64)
+        url="https://ci-mirrors.rust-lang.org/rustc/2018-04-02-sccache-x86_64-unknown-linux-musl"
+        ;;
+    aarch64)
+        url="https://ci-mirrors.rust-lang.org/rustc/2019-12-17-sccache-aarch64-unknown-linux-gnu"
+        ;;
+    *)
+        echo "unsupported architecture: $(uname -m)"
+        exit 1
+esac
 
+curl -fo /usr/local/bin/sccache "${url}"
```

Comment:
I'm thinking we might want to make sure that the sccache for aarch64 and x86_64 is always of the same version -- just to avoid potential collisions that aren't properly handled or whatever.

Reply:
Hmm, what do you have in mind to check that?

Reply:
Oh I just meant that we'd always have the same date for both -- not automatically checking it, but manually enforcing by updating both at the same time.

Reply:
We're not running this builder on CI yet, so there is not going to be duplication at least for a little while. Would you be ok if I follow up on sccache in a separate PR? Building it on musl is kinda of a pain, so I'm working on a script.